src/main.cc

```diff
@@ -47,7 +47,7 @@ int wmain(int argc, const wchar_t* argv[]) {
 
       unsigned short v1, v2, v3, v4;
       if (!parse_version_string(argv[++i], &v1, &v2, &v3, &v4))
-        return print_error("Unable to parse version string");
+        return print_error("Unable to parse version string for FileVersion");
```

Comment:
Also, I want to avoid allocating a new string just for errors, or augmenting `print_error` to take a variable number of arguments. The latter is way outside my comfort zone.